docs/readmes/howtos/inbound_roaming.md

```diff
@@ -192,12 +207,18 @@ Roaming Federation Networks configuration the following key
 That means that the Inbound FeG Network will be served by the local network
 (in this case called `example_feg_network`)
 
-### 5. Configure FeG Gateways
-If you have a Federated deployment remember to configure `s6a`, `gx`, `gy` on
-the gateway.
-
-Do the same for FeG gateway serving roaming subscribers, but just configure
-`s6a` and `s8`.
+### 5. Configure Roaming FeG Gateway
+Configure Roaming FeG gateway serving roaming subscribers, but just configure
+`s6a` and `s8`. Remember to configure local GTP port your PGW
```

Comment:
Configure the local GTP port so that your PGW knows where to send the requests?

I think you can omit `Remember to`